llama-index-integrations/llms/llama-index-llms-gemini/llama_index/llms/gemini/utils.py

```diff
@@ -41,6 +41,9 @@ def completion_from_gemini_response(
     raw = {
         **(type(top_candidate).to_dict(top_candidate)),
         **(type(response.prompt_feedback).to_dict(response.prompt_feedback)),
+        "usage_metadata": type(response.usage_metadata).to_dict(
+            response.usage_metadata
```

Comment:
Just checking, but will usage metadata always be there and be not `None` ?

Reply:
As far as I can see it is always present in the API responses. But I can make it failsafe to be sure 👍

Reply:
I adjusted it, have a look again